Remotion/Web/Development.WebTesting/ScreenshotCreation/BrowserContentLocators/FirefoxBrowserContentLocator.cs

```diff
@@ -60,13 +62,36 @@ public Rectangle GetBrowserContentBounds (IWebDriver driver)
       return result;
     }
 
+    [NotNull]
     private static AutomationElement GetFirefoxContentElement (AutomationElement firefoxWindow)
+    {
+      foreach (var _ in Enumerable.Range (0, 3))
```

Comment:
I started wondering again why we began using Enumerable.Range instead of a traditional for loop and came across [this](https://stackoverflow.com/questions/915745/thoughts-on-foreach-with-enumerable-range-vs-traditional-for-loop) discussion. I'm interested in @MichaelKetting 's opinion

Reply:
Imo it makes a loop where the index variable is unused easier to understand. A traditional for loop has much unneeded clutter around the range, which is the important part here - think of the range concept in other languages (pseudo: `for _ in 0..3`).

Reply:
hmm...since i grew up with C and Assembler, I'm used to seeing a for-loop in this case and wouldn't wonder about things when I see it. Enumerable.Range to me feels like extra overhead when reasoning on the code since it's not somehting that's in my normal vocabulary. Enumable.Range is just such a special case feature... Probably a lot to do with personal preferences.